usage/conf/log4j-cloud_usage.xml.in

```diff
@@ -48,7 +48,7 @@ under the License.
       </rollingPolicy>
 
       <layout class="org.apache.log4j.EnhancedPatternLayout">
-         <param name="ConversionPattern" value="%-5p [%c{3}] (%t:%x) (logid:%X{logcontextid}) %m%n"/>
+         <param name="ConversionPattern" value="%d{ISO8601} %-5p [%c{3}] (%t:%x) (logid:%X{logcontextid}) %m%n"/>
```

Comment:
@jburwell. RPM Conflict resolution will create .rpmnew log4j config (at least in case of RHEL/Centos). Existing installations will not be affected and if needed the change has to be manually merged to active log4j config.